pkg/scheduler/api/v1/types.go

```diff
@@ -158,6 +161,28 @@ type ExtenderArgs struct {
 	NodeNames *[]string `json:"nodenames,omitempty"`
 }
 
+// ExtenderPreemptionResult represents the result returned by preemption phase of extender.
+type ExtenderPreemptionResult map[string]*Victims
+
+// ExtenderPreemptionArgs represents the arguments needed by the extender to preempt pods on nodes.
+type ExtenderPreemptionArgs struct {
+	// Pod being scheduled
+	Pod *apiv1.Pod `json:"pod"`
+	// Victims map generated by scheduler preemption phase
+	NodeToVictims map[string]*Victims `json:"nodeToPods"`
+	// PodDisruptionBudget list
+	// +optional
+	Pdbs []*policy.PodDisruptionBudget `json:"pdbs,omitempty"`
```

Comment:
Is this all Pdbs configured? Seems excessive. Can we assume the extender gets this from apiserver?

Reply:
Yes. We can remove this PDBs if extender has ability to get this info.

Reply:
I agree with @ravigadde. Given our recent discussions and the expectations from extender to be able to get their required information from API server, I think we should remove this from the arguments.

Reply:
Yeah, the logic has removed this filed, just need to clean up here.